stestr/commands/load.py

```diff
@@ -68,11 +69,12 @@ def get_cli_help():
 
 def run(arguments):
     args = arguments[0]
+    stdout = open(os.devnull, 'w') if args.quiet else sys.stdout
```

Comment:
Heh, this is a bit hackier than what I was imagining. I was assuming just to not run the output bits at all if the flag was set, but I guess this is fine way to do it too. My only question is does this work on windows?

Reply:
I can answer the windows question for myself by reading the docs: https://docs.python.org/2/library/os.html#os.devnull

Reply:
heh, yeah, python3 as well. I think it should work.
https://docs.python.org/3/library/os.html#os.devnull